### PR TITLE
Problem: H1102NLT needs a 3D model

### PR DESCRIPTION
**Pull request by xylo04**
Solution: Add the STEP file, add that to the footprints, and refresh the footprints in the PCB

Screenshot 2024-08-14 at 07 56 24

### CODE REVIEW

**Comment by Vaelatern** [comment]
I'll need to review this one closer since the pcb itself is touched

**Comment by Vaelatern** [comment]
Can you please not do the rebuilt zone fills, and see if that gives me something I can merge nicely?

**Comment by xylo04** [comment]
Sure, I'll redo it that way.

File: bestboard-v2.kicad_pcb [hunk 1]
```diff
@@ -31624,7 +31932,18 @@
 			(net 138 "/TX3P")
 			(pinfunction "16")
 			(pintype "unspecified")
-			(uuid "71836d8d-c8c2-4910-a5f2-43d9b354318b")
+			(uuid "586ce7a7-1750-47de-ac2e-3a7e411b9cc0")
```
Comment (xylo04) [inline bestboard-v2.kicad_pcb:31935]: This appears to be the only relevant change regarding adding the 3D model.

Comment (Vaelatern) [reply bestboard-v2.kicad_pcb:31935]: Yeah. The font stuff must be a newer version of kicad I think.

**Pull request merged**